tests/ethereum/test_regressions.py

```diff
@@ -353,6 +353,40 @@ def test_mulmod(self):
         # 0x8000000000000000000000000000000000000000000000000000000082000011
         self.assertEqual(Z3Solver.instance().get_all_values(constraints, result), [2423129])
 
+    def test_related_to(self):
+        import gzip
+        from manticore import config
+        from manticore.core.smtlib.visitors import translate_to_smtlib
+        from manticore.core.smtlib import ConstraintSet, Z3Solver, Operators, BitVecConstant
+        import pickle, sys
+        filename = os.path.abspath(os.path.join(DIRPATH, "data", "ErrRelated.pkl.gz"))
```

Comment:
What's in this file?

Reply:
Also, should this test be in `ethereum` directory?

Reply:
My guess is that it's a state that exhibits the `get_related` bug, but pickled and gzipped so that we don't need to construct it from the source code.

Reply:
Yes. Not the state but the ConstraintSet that showed the problem. This is not really ethereum specific.

Reply:
Is it expensive to construct from source? A binary isn't easy to peruse over the web, for example.

Reply:
I saved the constraint and the constraintset that found to have trouble with old get_related.
I re build the constraint but the constraintset was big ish so I left it all binary and obscure.
Its a hack. I'll be much better to produce the testcases ourselves.
As it is still disabled by default I could remove this unintelligible test.